README.md

```diff
@@ -416,6 +416,7 @@ The `mst-gql` command currently accepts the following arguments:
 - `--modelsOnly` Generates only models, but no queries or graphQL capabilities. This is great for backend usage, or if you want to create your own root store
 - `--noReact` doesn't generate the React related utilities
 - `--force` When set, exiting files will always be overridden. This will drop all customizations of model classes!
+- `--namingConvention asis|js` When set to `js` it will generate type names and root collection names to conform to Javascript naming conventions, ie. type names will be PascalCased, and root collection names camelCased. The default (`asis`) value generates type names matching the original types names provided by the graphql schema
```

Comment:
what if instead of `--namingConvention js` we added a `--dontRenameModels` or similar option. This would make the default idiomatic while still having the option of using the values from the schema if wanted.

Reply:
That's OK with me. I'll add it soon.